dev/2018/spring/knights-to-kings/knights-to-kings.html

```diff
@@ -0,0 +1,196 @@
+                <li><strong>Hometown:</strong> Atlanta</li>
+                <li><strong>Degree:</strong> Human communication</li>
+              </ul>
+            </div>
+            <div class="col-md-4 col-xs-6">
+              <img src="[image filename='Evans-Thumbnail']" alt="Jordan Akins" class="img-circle img-responsive center-block">
+              <ul>
+                <li><strong>#66 Aaron Evans &rsquo;16</strong></li>
+                <li><strong>Position:</strong> Offensive lineman</li>
+                <li><strong>Hometown:</strong> Seffner, Florida</li>
+                <li><strong>Degree:</strong> Management</li>
+              </ul>
+            </div>
+            <div class="clearfix visible-sm visible-xs"></div>
+            <div class="col-md-4 col-xs-6">
+              <img src="[image filename='Franks-Thumbnail']" alt="Jordan Akins" class="img-circle img-responsive center-block">
+              <ul>
+                <li><strong>#15 Jordan Franks</strong></li>
+                <li><strong>Position:</strong> Tight end</li>
+                <li><strong>Hometown:</strong> Wakulla, Florida</li>
+                <li><strong>Degree:</strong> Integrated business</li>
+              </ul>
+            </div>
+            <div class="clearfix hidden-sm hidden-xs"></div>
+            <div class="col-md-offset-2 col-md-4 col-xs-6">
+              <img src="[image filename='Guerad-Thumbnail']" alt="Jordan Akins" class="img-circle img-responsive center-block">
+              <ul>
+                <li><strong>#93 Tony Guerad &rsquo;17</strong></li>
+                <li><strong>Position:</strong> Defensive lineman</li>
+                <li><strong>Hometown:</strong> Tampa, Florida</li>
+                <li><strong>Degree:</strong> Interdisciplinary studies</li>
+              </ul>
+            </div>
+            <div class="col-md-offset-0 col-md-4 col-xs-offset-3 col-xs-6">
+              <img src="[image filename='Lowry-Thumbnail']" alt="Jordan Akins" class="img-circle img-responsive center-block">
+              <ul>
+                <li><strong>#49 Seyvon Lowry &rsquo;17</strong></li>
+                <li><strong>Position:</strong> Defensive lineman</li>
+                <li><strong>Hometown:</strong> Jacksonville, Florida</li>
+                <li><strong>Degree:</strong> Interdisciplinary studies</li>
+              </ul>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+  <div class="container mt-2">
+    <div class="row">
+      <div class="col-md-8 col-md-offset-2 col-sm-10 col-sm-offset-1">
+        <span class="heading">Looking up from the bottom</span>
+        <p>"I hate remembering the 0-12 season, but it was a learning experience. We just had to keep on going. It helped all of us who were on that team that season. We know how it felt to lose every game. It was a big factor in winning every game this season." <span class="quoteby">&mdash; Tony Guerad</span></p>
```

Comment:
Individual quotes could probably be wrapped in `<blockquote>`s with `<cite>` elems for each `.quoteby`.